### PR TITLE
Fix CIRand admin function.

### PR DESCRIPTION
**Pull request by ogmage78**
* Trello card task - Extend ORM code to support multiple getLists or implement another method to query by alternate keys - re-implement cirand
* Many thanks to ddevec - he did the work on the ORM code revision.   I just tested and implemented for my use.
* Fixed undiscovered alignment bug at hook type we were sending as dword and it is a word.   That was shifting everything below it.
* Added priority to aceObject and set the aceObject priority field as a backing object for worldObject priority.
* Created function to calculate container burden.
* added view to world database that was needed for cirand and will be used in loot generation.
* updated position table to have a faster function for location per discord discussion.
* Updated trello board to reflect current project status.

### CODE REVIEW

File: Source/ACE/Entity/Container.cs [hunk 1]
```diff
@@ -72,14 +75,24 @@ public virtual void RemoveFromInventory(ObjectGuid inventoryItemGuid)
...
                 if (inventory.ContainsKey(inventoryItemGuid))
                     inventory.Remove(inventoryItemGuid);
             }
         }
 
+        public ushort UpdateBurden()
+        {
+            ushort calculatedBurden = 0;
+            lock (inventoryMutex)
+            foreach (KeyValuePair<ObjectGuid, WorldObject> entry in inventory)
```
Comment (Mogwai-TheFurry) [inline Source/ACE/Entity/Container.cs]: can we always do explicit scoping on a lock, please?
```
lock (inventoryMutex)
{
    // locked stuff here
}
```

File: Database/Updates/World/01-06-19-2017-alterPostionTable.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+ALTER TABLE `ace_world`.`ace_position`   
+  DROP COLUMN `landblock` ,
+  ADD COLUMN `landblock` INT(5) UNSIGNED AS ( ((landblockraw >> 16) & 0xFFFF)) VIRTUAL,
```
Comment (Mogwai-TheFurry) [inline Database/Updates/World/01-06-19-2017-alterPostionTable.sql]: so much easier to read...

File: Database/Updates/World/01-06-19-2017-alterPostionTable.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  DROP COLUMN `cell` ,
+  ADD COLUMN `cell` INT(5) UNSIGNED AS ( (CONV(RIGHT(LPAD(HEX(`landblockRaw`),8,'0'),4),16,10)) ) VIRTUAL,
```
Comment (Mogwai-TheFurry) [inline Database/Updates/World/01-06-19-2017-alterPostionTable.sql]: but this is so painful!  do we even have a use case for needing cell as a separate column?  if not, just drop it.

File: Source/ACE/Entity/Container.cs (hunk 1 above)
Comment (ogmage78) [inline Source/ACE/Entity/Container.cs]: Done

File: Database/Updates/World/01-06-19-2017-alterPostionTable.sql (hunk 3 above)
Comment (ogmage78) [inline Database/Updates/World/01-06-19-2017-alterPostionTable.sql]: Dropped ')

File: Database/Updates/World/01-06-19-2017-alterPostionTable.sql (hunk 2 above)
Comment (LtRipley36706) [inline Database/Updates/World/01-06-19-2017-alterPostionTable.sql]: Can also change back to smallint with this code
``landblock` smallint(5) GENERATED ALWAYS AS ((`landblockRaw` >> 16)) VIRTUAL,`
as the previous expression was forcing the change to an INT for whatever reason

**Pull request merged**